doc/reference/javascript_reference.rst

```diff
@@ -2131,6 +2131,12 @@ Relational fields
 
         <field name="category_id" widget="many2many_tags" options="{'color_field': 'color'}"/>
 
+    - no_edit_color: the name of a boolean field, which disable the possibility to change the color of the tags
```

Comment:
- no_edit_color: set to `True` to remove the possibility to change the color of the tags (default: `False`).